packages/iris-grid/src/sidebar/visibility-ordering-builder/sortable-tree/keyboardCoordinates.ts

```diff
@@ -0,0 +1,158 @@
+/* eslint-disable import/prefer-default-export */
+import {
+  closestCorners,
+  getFirstCollision,
+  KeyboardCode,
+  KeyboardCoordinateGetter,
+  DroppableContainer,
+} from '@dnd-kit/core';
+
+import type { SensorContext } from './types';
+import { getProjection } from './utilities';
+
+const directions: string[] = [
+  KeyboardCode.Down,
+  KeyboardCode.Right,
+  KeyboardCode.Up,
+  KeyboardCode.Left,
+];
+
+const horizontal: string[] = [KeyboardCode.Left, KeyboardCode.Right];
+
+export const sortableTreeKeyboardCoordinates: (
```

Comment:
Why not declare as `function`? Also probably should have some tests for this function.